LocalParks.Core/Domain/SportType.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace LocalParks.Core.Domain
+
+{
+    public enum SportType
```

Comment:
You could have these in a database table SportType with Columns Id and SportName, then you can add sports without having to do a redeploy of you app everytime.

Reply:
That makes sense, thanks, I add that in!